internal/commands/user.go

```diff
@@ -35,3 +37,21 @@ func getSlackUserInfo(
 
 	return &user, err
 }
+
+func getUsersInConversationParameters(
+	channelID string,
+	cursor string,
+) *slack.GetUsersInConversationParameters {
+
+	//Check if a cursor for a next page was received
+	if cursor != "" {
```

Comment:
:thinking: This was resolved here:
https://github.com/ResultadosDigitais/hellper/pull/80/files#diff-48f4e046d73ead02d5aeecd418f846e0R71